### PR TITLE
Update bleach to latest version

### LINKED ISSUE #1980: Listing page wrongly treats autolinks in description as unclosed HTML element
### Steps to reproduce the problem
1. Browse <https://snapcraft.io/featherpad>
1. Read the last line of the snap's description

### Expected behaviour
`</tsujan2000@gmail.com>` shouldn't appear

### Specs

- _URL:_ <https://snapcraft.io/featherpad>
- _Operating system:_ Ubuntu 19.04 AMD64
- _Browser:_ Mozilla Firefox 67.0.1

### PR DESCRIPTION
**Pull request by tbille**
# summary

Fixes #1980 

Previous version of bleach was closing everything that was considered as a tag. The update of bleach fixes this issue.

# qa

- `./run`
- http://0.0.0.0:8004/featherpad/preview
- http://0.0.0.0:8004/featherpad
- The email shouldn't be a closed tag at the end of the description

### CODE REVIEW

File: webapp/publisher/snaps/views.py [hunk 1]
```diff
@@ -876,8 +876,8 @@ def get_register_name_dispute():
 @login_required
 def post_register_name_dispute():
     try:
-        snap_name = flask.request.form.get("snap-name")
-        claim_comment = flask.request.form.get("claim-comment")
+        snap_name = flask.request.form.get("snap-name", "")
+        claim_comment = flask.request.form.get("claim-comment", "")
```
Comment (bartaz) [inline webapp/publisher/snaps/views.py]: What does this have to do with bleach on listing page?

Comment (tbille) [reply webapp/publisher/snaps/views.py]: we sanitize what is sent to the store
